examples/inverse/plot_covariance_whitening_dspm.py

```diff
@@ -50,10 +50,14 @@
 
 raw_fname = data_path + '/MEG/spm/SPM_CTF_MEG_example_faces%d_3D_raw.fif'
 
-raw = io.Raw(raw_fname % 1, preload=True)  # Take first run
+raw = io.Raw(raw_fname % 1)  # Take first run
+# To save time and memory for this demo, we'll just use the first
+# 2.5 minutes (all we need to get 30 total events) and heavily
+# resample 480->60 Hz (usually you woudln't do either of these!)
+raw.crop(0, 150., copy=False).load_data().resample(60, npad='auto')
```

Comment:
I ran it locally and verified the example looks the same, but it would be good to reconfirm